examples/server_affect_ignoring_policy.py

```diff
@@ -0,0 +1,16 @@
+"""Add an ignoring policy to all servers"""
+
+import os
+from configparser import ConfigParser
+from cbw_api_toolbox.cbw_api import CBWApi
+
+CONF = ConfigParser()
+CONF.read(os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', 'api.conf'))
+CLIENT = CBWApi(CONF.get('cyberwatch', 'url'), CONF.get('cyberwatch', 'api_key'), CONF.get('cyberwatch', 'secret_key'))
+
+INFO = {
+    "ignoring_policy": ""
```

Comment:
rename it into ignoring_policy_id and add a comment explaining how to find the possible ids